tests/unittests/GraphOptzTest.cpp

```diff
@@ -432,40 +440,43 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvButConvReused) {
 }
 
 TEST_F(GraphOptz, optimizeBatchNormAfterConvButVarReused) {
-  Node *A =
+  auto *A =
       mod_.createPlaceholder(ElemKind::FloatTy, {1, 10, 20, 3}, "A", false);
-  auto *filter =
-      mod_.createPlaceholder(ElemKind::FloatTy, {16, 5, 5, 3}, "filter", true);
-  auto *bias = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "bias", true);
-
-  ConvolutionNode *CV = F_->createConv(
-      "conv", A, filter, bias,
-      mod_.uniqueType(ElemKind::FloatTy, {1, 10, 20, 16}), 5, 1, 2, 1);
-  auto *beta = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "beta", true);
-  auto *gamma = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "gamma", true);
-
-  auto *mean = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "mean", false);
-  auto *var = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "var", false);
 
-  Node *BN = F_->createBatchNormalization("batch", CV, beta, gamma, mean, var,
-                                          3, 0.0001, 0.9);
-  SaveNode *ret = F_->createSave("ret", BN);
-  SaveNode *filterSave = F_->createSave("filterSave", CV->getFilter());
+  ConvolutionNode *CV = F_->createConv(bindings_, "conv", A, 16, 5, 1, 2, 1);
+  Node *BN =
+      F_->createBatchNormalization(bindings_, "batch", CV, 3, 0.0001, 0.9);
+  F_->createSave("ret", BN);
+  F_->createSave("filter", CV->getFilter());
 
   EXPECT_EQ(F_->getNodes().size(), 4);
+  optimizedF_ = optimizeFunction(F_);
+  ASSERT_EQ(A->getNumUsers(), 2);
+
+  auto *optimizedF_bn =
+      findFunctionNodeByName<SaveNode>(optimizedF_, "ret_save");
```

Comment:
Why is this suffixed with `_bn` if it is a `SaveNode`?

Reply:
fixed